tests/event-bus/e2e-tester/e2e-tester.go

```diff
@@ -545,6 +556,37 @@ func createEventActivation(subscriberNamespace string) error {
 	})
 }
 
+func createNamespace(name string) error {
+
+	ns := &apiv1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+			Labels: map[string]string{
+				"test": "test-event-bus",
+			},
+		},
+	}
+	err := retry.Do(func() error {
+		_, err := clientK8S.Core().Namespaces().Create(ns)
+		return err
+	}, retryOptions...)
+
+	if err != nil && !strings.Contains(err.Error(), "already exists") {
+		return fmt.Errorf("Namespace: %s could not be created: %v", name, err)
```

Comment:
> Error strings should not start with a capital letter because they'll often be prefixed before printing:

https://github.com/golang/go/wiki/Errors#errors

Reply:
changed